apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.html

```diff
@@ -7,9 +7,11 @@
         <dropdown-field [options]="months" (selectedOption)="getMonthStartAndEndDate($event)"
             [placeholder]="commonLocaleData?.app_select_month" [defaultValue]="selectedMonth?.label"></dropdown-field>
     </div>
-    <div class="input-group custom-datepicker custom-new-datepicker" (click)="showGiddhDatepicker($event)">
-        <input type="text" name="selectedDateRange" [value]="selectedDateRangeUi ? selectedDateRangeUi : ''"
-            class="giddh-datepicker date-range-picker border-radius-2" />
+    <div>
```

Comment:
what's the use of this div?

Reply:
Inside this date picker div will take all available space which causes datepicker show correct width but, its wrapper will take full space so if click on that div then also datepicker get opened, so as solution to this I wrapper that in side div